CodeChat/CommentDelimiterInfo.py

```diff
@@ -86,16 +86,10 @@
   'SQL':            ( 2,      2,            2),
   ##                  #,     <#,            #>
   'PowerShell':     ( 1,      2,            2),
-
-  # These languages have failing unit tests
-  # ---------------------------------------
-  ##                 //,     /*,            */
-  'Sass':           ( 2,      2,            2),
-
-  # These langauges have **NOT** been tested.
-  # -----------------------------------------
+  # ``/*`` ~ ``*/`` tests fail.
   ##                  #,     /*,            */
   'GAS':            ( 1,      2,            2),
+  # ``;`` tests fail.
```

Comment:
Since you've moved the GAS entry to the "These languages have failing unit tests" heading, delete it here. The same for the other duplicates (autohotkey, S, Clojure).